chain/penumbra/penumbra_chain.go

```diff
@@ -428,7 +428,7 @@ func (c *PenumbraChain) start(ctx context.Context) error {
 		eg.Go(func() error {
 			return n.TendermintNode.CreateNodeContainer(
 				egCtx,
-				fmt.Sprintf("--proxy-app=tcp://%s:26658", n.PenumbraAppNode.HostName()),
+				fmt.Sprintf("--proxy_app=tcp://%s:26658", n.PenumbraAppNode.HostName()),
```

Comment:
does this also need the conditional separator like you did for the config?

Reply:
Yes! indeed it does. Fixed.